src/nft/components/collection/PriceRange.tsx

```diff
@@ -33,52 +49,134 @@ export const PriceRange = () => {
   }
 
   return (
-    <Row gap="12" marginTop="12" color="textPrimary">
-      <Row position="relative" style={{ flex: 1 }}>
-        <NumericInput
-          style={{
-            width: isMobile ? '100%' : '142px',
-            border: '2px solid rgba(153, 161, 189, 0.24)',
-          }}
-          borderRadius="12"
-          padding="12"
-          fontSize="14"
-          color={{ placeholder: 'textSecondary', default: 'textPrimary' }}
-          backgroundColor="transparent"
-          placeholder="Min"
-          defaultValue={minPrice}
-          onChange={(v: FormEvent<HTMLInputElement>) => {
-            scrollToTop()
-            setMinPrice(v.currentTarget.value)
-          }}
-          onFocus={handleFocus}
-          value={minPrice}
-          onBlur={handleBlur}
-        />
+    <TraitsHeader title="Price range" index={-9}>
```

Comment:
I would prefer not to change logic in this file since it works when I tested.  Any suggested logic changes, I would ask to please test thoroughly, thanks